backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/repos/ApplicationRepository.kt

```diff
@@ -199,4 +199,22 @@ object ApplicationRepository {
                 .sortByKeys({ it.id.value }, ids)
         }
     }
+
```

Comment:
So these Repository.kt files are businesslogic mixed with database access?

Reply:
I think mostly access to DB / persistence layer. There should not be a lot of businesslogic